tests/test_cmd.py

```diff
@@ -6,40 +6,20 @@
 import numpy as np
 import os
 import subprocess
+import time
 
-os.chdir(os.path.join(os.path.dirname(os.path.abspath(__file__)), '..'))
-outfile = 'tests/output.txt'
-
-
-def setup_function(function):
-    try:
-        os.remove(outfile)
-    except OSError:
-        pass
-
-
-teardown_function = setup_function
-
```

Comment:
This is needed so we don't leave files hanging around.  Did you find another way to do this that I didn't notice?

Reply:
probably do need, see comment below